cmd/mdatagen/templates/readme.md.tmpl

```diff
@@ -1,17 +1,20 @@
 <!-- status autogenerated section -->
-{{- if ne .Status.Stability "" }}
+{{- if len .Status.Stability }}
 | Status                   |           |
 | ------------------------ |-----------|
-| Stability                | [{{ .Status.Stability }}]   |
-{{- if ne .Status.Class "extension" }}
-| Supported pipeline types | {{ stringsJoin .Status.Pipelines ", " }}   |
+{{- $class := .Status.Class }}
+{{- $idx := 0 }}
+{{- range $stability, $value := .Status.Stability }}
+| {{ if not $idx }}Stability{{- end }}                | [{{ $stability }}] {{ if ne $class "extension" }}[{{ stringsJoin $value ", " }}] {{ end }}  |
+{{- $idx = inc $idx }}
 {{- end }}
 | Distributions            | [{{ stringsJoin .Status.Distributions "], [" }}] |
 {{- if .Status.Warnings }}
 | Warnings                 | [{{ stringsJoin .Status.Warnings ", " }}](#warnings) |
```

Comment:
nit to keep at least the first column aligned :)
```suggestion
| Distributions | [{{ stringsJoin .Status.Distributions "], [" }}] |
{{- if .Status.Warnings }}
| Warnings      | [{{ stringsJoin .Status.Warnings ", " }}](#warnings) |
```

Reply:
Done